kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -337,6 +341,63 @@ redirect cmd path = do
     redirectToFile :: String -> IO ()
     redirectToFile = writeFile path
 
+tryAxiomClaim
+    :: forall level
+    .  MetaOrObject level
+    => level ~ Object
+    => Either AxiomIndex ClaimIndex
+    -> ReplM level ()
+tryAxiomClaim eac = do
+    ReplState { axioms, claims, claim, graph, node, stepper } <- get
+    case getAxiomOrClaim axioms claims node of
```

Comment:
I don't really like having a `case` here, but I guess there is not error handling built into `ReplM`?

Reply:
Yeah, there wasn't a need for that, but perhaps adding it might make some things simpler. The answer to this should be easier to reach once the REPL gets more features.